vdr/doc/resolvers_test.go

```diff
@@ -190,30 +190,46 @@ func TestResolver_Resolve(t *testing.T) {
 	})
 
 	t.Run("docA is controller of docB and docA is deactivated", func(t *testing.T) {
-		ctrl := gomock.NewController(t)
-		store := types.NewMockStore(ctrl)
-		resolver := Resolver{Store: store}
-		store.EXPECT().Resolve(*id456, resolveMD).Return(&docB, &types.DocumentMetadata{}, nil)
-		store.EXPECT().Resolve(*id123, resolveMD).Return(&docA, &types.DocumentMetadata{}, nil)
-
-		doc, _, err := resolver.Resolve(*id456, resolveMD)
-
-		assert.Error(t, err)
-		assert.Equal(t, types.ErrNoActiveController, err)
-		assert.Nil(t, doc)
-	})
-
-	t.Run("docA is controller of docB and docA is deactivated but is allowed", func(t *testing.T) {
```

Comment:
this is still a valid test?

Reply:
Yes, it still there but now moved to line 221.